### DISCUSSION TITLE
Another approach to access OPC data with Python

### DISCUSSION THREAD

**Discussion by duduyoyo** [FreeOpcUa/opcua-asyncio#1028] [General]
Check [here](https://github.com/duduyoyo/WebSocket4OPC), an alternative to use Python to retrieve OPC data easily and straightforwardly. Assuming you have a classic OPC server installed, this approach will cover all the others like authentication, secure connection, firewall etc. Simply install and launch it within minutes without checking any thousands page document. You have absolutely ZERO learning curve since HTTP/JSON is its intuitive language, no DCOM period. Give it a try and it could blow your mind away.

**Comment by AndreasHeine** [comment]
this is an opc ua library not opc classic!!!

**Comment by oroulet** [comment]
that was the strangest message ever in that forum!
